tap_hubspot/hubspot.py

```diff
@@ -24,8 +22,7 @@ def __init__(
         self,
         config: Dict,
         tap_stream_id: str,
-        start_date: datetime,
-        end_date: datetime,
+        event_state: DefaultDict[Set, str],
```

Comment:
Two questions:
1. Why use DefaultDict
2. What is the first argument of DefaultDict, and why is it `Set` in this case. Could you provide me some examples of how `event_state` looks like?

Reply:
Does that mean that key is string and value is a `Set`?

Reply:
```event_state={"contact_start_date": str, "contact_end_date": str, "event_contact_ids": set,"hs_calculated_form_submissions" : set}``` `event_state` is set when syncing `contacts` table. the `events_contacts` table is synced according to 
 `event_contact_ids`, `contact_start_date`, `contact_start_date.`    `submissions` table is synced according to `hs_calculated_form_submissions`.